protocol/messenger.go

```diff
@@ -1296,8 +1292,8 @@ func (m *Messenger) RemoveMemberFromGroupChat(ctx context.Context, chatID string
 	chat.updateChatFromGroupMembershipChanges(group)
 
 	response.AddChat(chat)
```

Comment:
Looking at number of same edits in different placers, seems that this actions sequence can benefit from being in a separate function:
```
reponse.AddChat()
reponse.AddMessages()
m.persistence.SaveMessages()
m.saveChat()
```